test/es-module/test-esm-live-binding.mjs

```diff
@@ -0,0 +1,13 @@
+// Flags: --experimental-modules
+
+import '../common';
+import assert from 'assert';
+
+import fs, { readFile } from 'fs';
+
+const s = Symbol();
+
+fs.readFile = () => s;
+
+assert.strictEqual(fs.readFile(), s);
+assert.strictEqual(readFile(), s);
```

Comment:
Still need to test:

- `delete`
- `delete` and `set` afterwards
- `set`
- `defineProperty`
- All of the above, but with accessor properties